examples/image.rs

```diff
@@ -48,7 +48,7 @@ fn main() -> Result<(), PdfiumError> {
             .pages()
             .first()?
             .render_with_config(
-                &PdfRenderConfig::new().set_target_width(target_pixel_width_of_bitmap),
+                &PdfRenderConfig::new().set_target_width(target_pixel_width_of_bitmap.into()),
```

Comment:
bonus change - but was needed to get things to compile

i.e. when running `cargo test` on master:

```
   Compiling pdfium-render v0.8.1 (/Users/bduffield/Git/pdfium-render)
error[E0308]: mismatched types
   --> examples/image.rs:51:58
    |
51  |                 &PdfRenderConfig::new().set_target_width(target_pixel_width_of_bitmap),
    |                                         ---------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `i32`, found `u16`
    |                                         |
    |                                         arguments to this method are incorrect
    |
note: associated function defined here
   --> /Users/bduffield/Git/pdfium-render/src/render_config.rs:179:12
    |
179 |     pub fn set_target_width(mut self, width: Pixels) -> Self {
    |            ^^^^^^^^^^^^^^^^
help: you can convert a `u16` to an `i32`
    |
51  |                 &PdfRenderConfig::new().set_target_width(target_pixel_width_of_bitmap.into()),
    |                                                                                      +++++++

For more information about this error, try `rustc --explain E0308`.
error: could not compile `pdfium-render` due to previous error
warning: build failed, waiting for other jobs to finish...
```